index.html

```diff
@@ -4,6 +4,10 @@
   <title>YOUR NAME HERE</title>
 </head>
 <body>
+  <h1>Adam Reid</h1>
+  <img>https://media.giphy.com/media/gU25raLP4pUu4/giphy.gif</>
```

Comment:
I intended to make a new .html file called Adam.html. Are we only meant to have just one .html file?